StatsdClient/StatsdClientExtensions.cs

```diff
@@ -14,7 +14,14 @@ public static class StatsdClientExtensions
         /// <param name="count">The counter value (defaults to 1).</param>
         public static void LogCount(this IStatsd client, string name, int count = 1)
         {
-            client.LogCountAsync(name, count).Wait();
+            try
```

Comment:
Can you revert this change, the aggregate exception should be boiled up as it can contain multiple exceptions.